packages/app-project/pages/_app.js

```diff
@@ -2,7 +2,7 @@ import { Box } from 'grommet'
 import makeInspectable from 'mobx-devtools-mst'
 import { Provider } from 'mobx-react'
 import Error from 'next/error'
-import React, { useEffect, useMemo } from 'react'
+import { useEffect, useMemo } from 'react';
```

Comment:
[minor][code style] During the refactor, a bunch of semicolons `;` have been added by accident. I think there are about 22 instances of this.

Reply:
Oh, I think the React codemod must have done that.